sdk/recoveryservices/Azure.ResourceManager.RecoveryServices/src/Generated/Extensions/SubscriptionResourceExtensionClient.cs

```diff
@@ -41,6 +47,70 @@ private string GetApiVersionOrNull(ResourceType resourceType)
             return apiVersion;
         }
 
+        /// <summary>
+        /// API to get details about capabilities provided by Microsoft.RecoveryServices RP
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/providers/Microsoft.RecoveryServices/locations/{location}/capabilities</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>RecoveryServices_Capabilities</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="location"> Location of the resource. </param>
+        /// <param name="input"> Contains information about Resource type and properties to get capabilities. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<CapabilitiesResponse>> CapabilitiesRecoveryServiceAsync(AzureLocation location, ResourceCapabilities input, CancellationToken cancellationToken = default)
```

Comment:
The method requires a verb, like `GetCapabilitiesRecoveryService` or `FetchCapabilitiesRecoveryService`. Looks like the root cause is due to the `operation_id` in the swagger definition. Use `override-operation-name` to give it a good name. See here for example: https://github.com/Azure/azure-sdk-for-net/blob/3a7060001946c49af98370638aa7fffadca4c905/sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/autorest.md?plain=1#L58-L60